tests/common/accuracy_control/test_algorithm.py

```diff
@@ -0,0 +1,250 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from dataclasses import dataclass
+from unittest.mock import Mock
+
+import numpy as np
+import pytest
+
+from nncf.common.graph.graph import NNCFNode
+from nncf.quantization.algorithms.accuracy_control.algorithm import QuantizationAccuracyRestorer
+from nncf.quantization.algorithms.accuracy_control.algorithm import QuantizationAccuracyRestorerReport
+from nncf.quantization.algorithms.accuracy_control.algorithm import _create_message
+from tests.common.accuracy_control.backend import AABackendForTests
+from tests.common.quantization.mock_graphs import get_mock_model_graph_with_mergeable_pattern
+
+
+def test_create_message():
+    nodes = [
+        NNCFNode(
+            {
+                NNCFNode.NODE_NAME_ATTR: "node_name_1",
+            }
+        ),
+        NNCFNode(
+            {
+                NNCFNode.NODE_NAME_ATTR: "node_name_2",
+            }
+        ),
+    ]
+    result = _create_message(nodes)
+    assert isinstance(result, str)
+    assert result == "\n".join(["\t" + node.node_name for node in nodes])
```

Comment:
Try to obtain expected values without adding any additional code wherever possible. It allows for quickly understanding the expected behavior of the method being tested.
```suggestion
    assert result == "\tnode_name_1\n\tnode_name_2"
```